context-free.md

```diff
@@ -2,21 +2,54 @@
 
 ##  Who is this programming language for?
 
+This programming language is for people who want to create art or 
+images on the computer using code. 
 
 ## What is easy to do in this language? Why is it easy?
 
+Of course, it is very easy to create images. More specifically, it is easy to 
+create images with repeating components.  By defining rules, it is very easy to 
+create individual shapes which can be called and used in other parts of the image. 
+Furthermore, since users can call the rules in other parts of the code,
+recursion is relatively easy and shapes can be reused in different variations 
+again easily. Though saying it is easy to create images is very broad, what 
+comes to mind is how much cleaner it is to read in comparison to SVG, which is 
```

Comment:
I think SVG is close to the DSL side of the spectrum, since it does serve a very specific domain: creating images.  Otherwise, it is pretty limited in what it can do.